extensions.json

```
@@ -898,6 +898,12 @@
       "repository": "https://github.com/mikestead/vscode-dotenv",
       "version": "1.0.1"
     },
+    {
+      "id": "mindaro-dev.file-downloader",
+      "repository": "https://github.com/microsoft/vscode-file-downloader",
+      "version": "1.0.10",
+      "checkout": "master"
```

This repository has no GitHub Releases or dedicated release tag, so there is no need to try and pin or checkout a specific version.

The best we can do here is publish straight from the default branch.

```suggestion
      "repository": "https://github.com/microsoft/vscode-file-downloader"
```